src/modules/site-v2/templates/_scripts/utils.js

```diff
@@ -44,3 +44,28 @@ function form_data_to_object(form_id) {
   });
   return data;
 }
+
+
+function force_download(el, url) {
+
+  // If this function has run on the element already, use the href normally
+  if (el.href[el.href.length - 1] != '#') return;
+
+  // Fetch the provided URL and create a blob object from it
+  fetch(url)
```

Comment:
a caveat of having this be part of an event handler, is that if there are any issues fetching the blob (eg: cors, missing auth, etc), there is no UI warning for the end-user. How can we expose error-states to the front-end to avoid the end-user running into the situation "clicked on a button/link and nothing happened"?

Reply:
Currently, I don't think we have any way to signal a failed download to the user... might be worth looking into for more than just this link.

(If the file doesn't exist in GCP, for example, the user gets a `NoSuchKey` XML error page.)